packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -348,9 +406,12 @@ export const prepareAuctionBook = (baggage, zcf) => {
             (a, b) => compareValues(a[1].seqNum, b[1].seqNum),
           );
 
+          const { totalToRaise } = this.state;
           const { helper } = this.facets;
           for (const [key, { seat, price: p, wanted }] of prioritizedOffers) {
-            if (seat.hasExited()) {
+            if (totalToRaise && AmountMath.isEmpty(totalToRaise)) {
```

Comment:
this is the line that made me think the vars should have `limit`, because that's the condition.
```suggestion
            if (collectionLimit && AmountMath.isEmpty(collectionLimit)) {
```

Reply:
`goal` is better than `limit` because the auction might end up with more or less, but that's a number to aim for.